Automattic/wp-calypso
# Migration: Improve source site input field

Fixes https://github.com/Automattic/wp-calypso/issues/87455

## Proposed Changes

* These changes will trim the migration source site input field to prevent friction/error messages from displaying when a user mistakenly adds whitespaces before or after their source site URL.

## Testing Instructions



* Checkout branch or use live link
* Navigate to `/sites?hosting-flow=true` and click the `Migrate a site` button.
* Test the input field to ensure it trims whitespaces and continues to function correctly when you click `Continue`

## Pre-merge Checklist
